definitions/divorce/json/CaseEvent/CaseEvent-general-referral-nonprod.json

```diff
@@ -23,5 +23,17 @@
     "PostConditionState": "GeneralConsiderationComplete",
     "SecurityClassification": "Public",
     "ShowSummary": "Y"
+  },
+  {
+    "LiveFrom": "20/10/2020",
+    "CaseTypeID": "DIVORCE",
+    "ID": "GeneralReferralPayment",
+    "Name": "General Referral Payment",
+    "Description": "General Referral Payment",
+    "DisplayOrder": 18,
+    "PreConditionState(s)": "*",
+    "PostConditionState": "AwaitingGeneralConsideration",
+    "CallBackURLAboutToStartEvent": "${CCD_DEF_COS_URL}/set-up-confirm-service-payment",
```

Comment:
are you sure this endpoint is correct?

I mean - it populates field `generalApplicationWithoutNoticeFeeSummary`. Are you going to use the very same field that is used by Service Application's events?

Reply:
As far as im aware, this is the correct endpoint to call to. We reuse the same payment flow as before and this is where it points to to populate

Reply:
but you have field of type SummaryOrder defined in other file and I have a feeling we should use that field, shouldn't we?

Reply:
I agree. i have implimented it now and it is using the existing field that will work with this endpoint